aviary/mission/gasp_based/ode/flight_path_ode.py

```diff
@@ -216,33 +196,33 @@ def setup(self):
         self.add_excess_rate_comps(nn)
 
         # Example of how to use a print_comp
-        if False:
-            from aviary.utils.functions import create_printcomp
-            dummy_comp = create_printcomp(
-                all_inputs=[
-                    Dynamic.Mission.DISTANCE,
-                    Dynamic.Mission.THROTTLE,
-                    Dynamic.Mission.THRUST_TOTAL,
-                    'required_thrust',
-                    Dynamic.Mission.ALTITUDE,
-                    'load_factor',
-                    'required_lift',
-                    Dynamic.Mission.MASS,
-                    Dynamic.Mission.FLIGHT_PATH_ANGLE,
-                    'alpha',
-                ],
-                input_units={
-                    'required_thrust': 'lbf',
-                    'required_lift': 'lbf',
-                    'alpha': 'deg',
-                    Dynamic.Mission.FLIGHT_PATH_ANGLE: 'deg',
-                })
-            self.add_subsystem(
-                "dummy_comp",
-                dummy_comp(),
-                promotes_inputs=["*"],)
-            self.set_input_defaults(
-                Dynamic.Mission.DISTANCE, val=0, units='NM')
+        # if False:
```

Comment:
I would just remove this too.

Reply:
This is here as an example of how a print_comp can be be added for debugging. The `if False` keeps it from printing when it isn't needed. We could change this to something like `if verbosity >= verbose`